tests/test_alignment.py

```diff
@@ -30,7 +30,7 @@ def _create_wcs_object_without_distortion(
     shape,
 ):
     # subtract 1 to account for pixel indexing starting at 0
-    shift = models.Shift() & models.Shift()
+    shift = models.Shift(0) & models.Shift(0)
```

Comment:
Why was this change necessary?  Does this shift actually subtract 1 as the comment indicates?

Reply:
Oh, this was a leftover from testing

Reply:
The comment is incorrect either way. The default value for the shift is 0 anyway. So my change does not have any effect. I'll undo it.